### DISCUSSION TITLE
How to outline Seurat clusters in black?

### DISCUSSION THREAD

**Discussion by yamihn** [satijalab/seurat#8728] [Q&A]
Hello everyone,

I'm currently working with Seurat for single-cell RNA-seq analysis, and I'm trying to enhance the visualization of my UMAP plots. I was wondering if there's a way to outline the clusters identified by Seurat in black. I've tried using the borders = TRUE and border.color = "black" parameters in the DimPlot() function, but it seems they're not supported.

Does anyone know if there's a way to achieve this within Seurat or with additional customization using ggplot2?

Any help or suggestions would be greatly appreciated.

Thank you!

**Comment by saumyapo** [comment]
I have the same question. Did you find a solution?
